automate/serializers.py

```diff
@@ -39,4 +40,45 @@ def validate(self, attrs):
         owner = self.context.get("owner")
         attrs["owner"] = owner
         # TODO: Validate the owner/token/repo name are all correct and can connect to Repository
+        # TODO: I don't understand the todo above
```

Comment:
@timizuoebideri1 the todo above. I don't quite understand it

Reply:
After a user adds a Project, Its possible that the token/owner/repo is not correct. So we test to verify we can connect to their repo; 
If it fails raise a validationError